components/Card/CardComponents/HotelDetails.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="flex w-full">
+    <div
+      :class="
+        showAmenities
+          ? 'description-section w-3/5'
+          : 'description-section w-full'
+      "
+    >
+      <p class="hotel-desc font-robotoRegular text-xs text-white">
+        {{ description }}
+      </p>
+    </div>
+
+    <div
+      v-if="showAmenities"
```

Comment:
Check whether amenities array exists and if the length is not 0
`amenities?.length` in v-if